packages/tasks/src/tasks/image-feature-extraction/spec/input.json

```diff
@@ -0,0 +1,26 @@
+{
+	"$id": "/inference/schemas/image-feature-extraction/input.json",
+	"$schema": "http://json-schema.org/draft-06/schema#",
+	"description": "Inputs for Image Feature Extraction",
+	"title": "ImageFeatureExtractionInput",
+	"type": "object",
+	"properties": {
+		"inputs": {
+			"description": "The input image data",
+			"type": "string"
```

Comment:
```suggestion
```

Reply:
Same type of input as other image tasks https://github.com/merveenoyan/huggingface.js/blob/add_ift_spec/packages/tasks/src/tasks/image-classification/spec/input.json#L8